### PR TITLE
Latte: inference scripts and training on Sky TimeLapse

### PR DESCRIPTION
**Pull request by wtomin**
# What does this PR do?

1. The ckpt conversion scripts for vae and latte checkpoint;
2. The inference scripts for latte using sky-timelapse pretrained ckpt and ucf-101 pretrained ckpt;
3. The training dataset for sky timelapse dataset;
4. The training script of latte on the sky timelapse: the loss on Ascend is decreasing. The ckpt at epoch=150 generates reasonable videos.

TODOs:
1. To support more datasets for training;
2. Speed acceleration (Flash-Attention is not working correctly! Needs to be fixed).

### CODE REVIEW

File: examples/latte/README.md
Comment (SamitHuang) [inline examples/latte/README.md:1]: can we append some training results as a reference?

File: examples/dit/diffusion/gaussian_diffusion.py [hunk 1]
```diff
@@ -149,7 +149,12 @@ def p_mean_variance(self, model, x, t, clip_denoised=True, denoised_fn=None, mod
         if model_kwargs is None:
             model_kwargs = {}
 
-        B, C = x.shape[:2]
+        if x.dim() == 4:
```
Comment (zhtmike) [inline examples/dit/diffusion/gaussian_diffusion.py:152]: better to decouple from dit module

Comment (wtomin) [reply examples/dit/diffusion/gaussian_diffusion.py]: Fixed.

File: examples/latte/README.md
Comment (wtomin) [reply examples/latte/README.md:1]: Training results have not been ready yet.

**Review by zhtmike — APPROVED** [review]

**Review by hadipash — APPROVED** [review]

File: mindone/models/latte.py [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+    xavier_uniform_,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class Latte(nn.Cell):
+    def __init__(
+        self,
+        input_size=32,
```
Comment (hadipash) [inline mindone/models/latte.py:26]: Since this part is being integrated into the library, it would be nice for the code to be well typed and documented.

Comment (wtomin) [reply mindone/models/latte.py:44]: Thanks for your advice. It is improved.
